CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,81 @@
+# Up-For-Grabs Code of Conduct
+
+This document outlines how contributors to the Up-For-Grabs project
+should interact with each other, and the process for resolving issues.
+
+Projects listed on Up-For-Grabs are not covered by this code of conduct.
+Inquiries about a specific project should be addressed according to that
+project's governance rules.
+
+## Our Pledge
+
+In the interest of fostering an open and welcoming environment, we as
+contributors and maintainers pledge to making participation in our project and
+our community a harassment-free experience for everyone, regardless of age, body
+size, disability, ethnicity, gender identity and expression, level of experience,
+nationality, personal appearance, race, religion, or sexual identity and
+orientation.
+
+## Our Standards
+
+Examples of behavior that contributes to creating a positive environment
```

Comment:
```suggestion
Behaviors that contribute to creating a positive environment
```

tiny nitpick suggestion on wording.

Reply:
If you run the caveat on the one below, this doesn't appear to mean the same thing as what it replaces.
Not that it wasn't badly written in the first place, but alas.